2022Swerve/src/main/java/frc/robot/hardware/MotorController.java

```diff
@@ -0,0 +1,76 @@
+package frc.robot.hardware;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import frc.robot.Constants.MotorDefaults;
+
+public class MotorController {
+    public static enum MotorControllers {
```

Comment:
Consider a different name for the enum?  Maybe config?  so that you can include MotorController.config into the subsystems?

Reply:
done